### ISSUE TITLE
unable to install multiple errors

### ISSUE THREAD

**Issue by spectroman** [kubevious/kubevious#83]
Unable to install due to multiple errors

kubectl create ns kubevious
helm install kubevious . -n kubevious --wait --atomic

Environment Details:
 - Kubernetes: 1.24.2 / kubeadm / baremetal

The first thing to notice is that mysql por never gets ready because readiness probe gets stuck trying to execute mysql `select 1`

To solve the readiness probe I have changed to:
```
command: ["mysql", "-ukubevious", "-pkubevious", "-h","127.0.0.1", "-e","SELECT 1;"]
```
I'm not sure why, but the variables are not working there.

Next, after Mysql is ready, pods backend, collector and guard never get ready with the error:

At guard:
```
Readiness probe failed: Get "http://10.209.76.188:4004/": dial tcp 10.209.76.188:4004: connect: connection refused 
```
At collector log
![image](https://github.com/kubevious/kubevious/assets/8914033/bafdd0f6-fca0-46e1-be2c-6cd438e1d580)

The logs from the 3 failing pods are similar, but I see a database error at the collector...

Any one has any ideas? Maybe there is something im missing?

**Comment by rubenhak** [comment]
Hi, we need to see complete MySQL logs and check other events that coulc cause MySQL not coming up. This could also be cause by CSI not getting able to allocate a storage volume. If MySQL doesn't come up the the backend would stay unhealthy and K8s would restart it.

**Comment by rubenhak** [comment]
Something is off with the MySQL; that's why you see it restarting every minute. 

Maybe you can try installing MySQL separately using the Bitnami chart: [https://bitnami.com/stack/mysql/helm](https://bitnami.com/stack/mysql/helm), then install Kubevious Dashboard using external mysql instance. You would need to set **mysql.external.xxx** values. See configuration here: [https://github.com/kubevious/helm#%EF%B8%8F-configuration](https://github.com/kubevious/helm#%EF%B8%8F-configuration)

**Comment by rubenhak** [comment]
The main root repo doesn't contain any code. Here is the [High Level Architecture](https://github.com/kubevious/kubevious/blob/main/ARCHITECTURE.md) and [Code Repos](https://github.com/kubevious/kubevious/blob/main/ARTIFACTS.md).

You could be hitting the scale limits and increasing CPU/Memory requests on Mysql (the bitnami chart), Parser and Collector could help. Yet you chose to complain instead of giving any meaningful information. See here for [helm configuration](https://github.com/kubevious/helm#-troubleshooting).

Also can reduce the load by excluding certain resources using **kubevious.api.skipEvents** and **kubevious.api.skipped**. Details [here](https://github.com/kubevious/helm#%EF%B8%8F-configuration).

You can also check out the CLI project, which doesn't have those scale limits: [https://github.com/kubevious/cli](https://github.com/kubevious/cli).

Maybe you're right that this project is not for you. I don't know, cause you didn't tell much. But what I can tell is that it doesn't seem like you've been involved with open-source communities. This is not now things work. No one has obligation to help, figure out problems you have in the environment and hear about you "wasting your time". If you want someone to spend their time to help you, you have to be respectful. Hope you understand.

**Comment by spectroman** [comment]
So @rubenhak I am sorry you felt offended, when somehow when I admitted my failure to configure what apparently is your powerful software.

When I say, and I quote: `So either I don't understand (could be likely) what should happen here or it just doesn't work...` its a comment on my ability and on some differences on my cluster that made Kubevious not run innit in an "out-of-the-box" state. Meaning, it has nothing to do with you, your abilities or your ego. I don't know you, I am talking about me and my experience trying to work this out. If my way of thinking bothers you, well, I am sorry, but news flash, no one in the entire world thinks like you cos we are all different. Thanks nature for that.

Now when growing up, I had my education and I learned that one of the parts that doesn't fit in social interaction is the intention to offend others, even so slightly indirectly, if you took that meaning I am sorry, and obviously I might have attracted your unwarranted wrath to judge me and my experience without have any idea about who I am, what I do or anything else. If that is just your way to defend your self from some conversation you felt attacked from, well, I can be sorry for that too.

Now I could just give you a litany of reasons on why your behavior is not healthy for the open-source community but I will just leave at this...

By the way, I didn't really gave up on trying to configure Kubevious, cos I really found this absolutely interesting, I just got the sense that I don't want to eat up much of time from people who might be more busy than helping someone who can help himself, if I go through and read the documentation more carefully I might find "reasons" and I am sure that the brilliant ppl who made this have all the answers in there, in the meanwhile the "issue" might be a non issue, so why continue?

The thing I wanted to comment in there, which might have frustrated me a bit, is the interface talking about the Agent and I could not find a direct documentation about an agent, so that was weird... but again, it is just the speed I was expecting to make this work... so I will just leave it there for a while and I will come back with more time on it...

In the mean time man... peace.
